addons/website_slides/static/src/js/components/course_join_popup.js

```diff
@@ -0,0 +1,16 @@
+import { Component } from "@odoo/owl";
+
+export class SlidesCourseJoinPopup extends Component {
```

Comment:
Shouldn't this, and WebsiteSlidesCourseQuizQuestionForm, be "public" components (in ./public/components, with their template next to them?

Reply:
~~Not resolved.~~ My bad, it was.